extensions/wikia/ArticleVideo/styles/ooyala/base/_reset.scss

```diff
@@ -0,0 +1,242 @@
+// Reset
+//
+// Global resets to common HTML elements and more for easier usage by html5-skin.
+// Adds additional rules on top of Normalize.css, including several overrides.
+
+
+// Reset the box-sizing
+//
+// Change from `box-sizing: content-box` to `border-box` so that when you add
+// `padding` or `border`s to an element, the overall declared `width` does not
+// change. For example, `width: 100px;` will always be `100px` despite the
+// `border: 10px solid red;` and `padding: 20px;`.
+//
+// Credit: https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/
+// https://github.com/twbs/bootstrap/blob/v4-dev/scss/_reboot.scss
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+
+// Make viewport responsive
+//
+// @viewport is needed because IE 10+ doesn't honor <meta name="viewport"> in
+// some cases. See http://timkadlec.com/2012/10/ie10-snap-mode-and-responsive-design/.
+// Eventually @viewport will replace <meta name="viewport">. It's been manually
+// prefixed for forward-compatibility.
+//
+// However, `device-width` is broken on IE 10 on Windows (Phone) 8,
+// (see http://timkadlec.com/2013/01/windows-phone-8-and-device-width/ and https://github.com/twbs/bootstrap/issues/10497)
+// and the fix for that involves a snippet of JavaScript to sniff the user agent
+// and apply some conditional CSS.
+//
+// See http://getbootstrap.com/getting-started/#support-ie10-width for the relevant hack.
+
+
+// Wrap `@viewport` with `@at-root` for when folks do a nested import (e.g.,
+// `.class-name { @import "bootstrap"; }`).
+//@at-root {
+//  @-moz-viewport      { width: device-width; }
+//  @-ms-viewport       { width: device-width; }
+//  @-o-viewport        { width: device-width; }
+//  @-webkit-viewport   { width: device-width; }
+//  @viewport           { width: device-width; }
+//}
+
+
+//
+// Typography
+//
+
+// Remove top margins from headings
+//
+// By default, `<h1>`-`<h6>` all receive top and bottom margins. We nuke the top
+// margin for easier control within type scales as it avoids margin collapsing.
+h1, h2, h3, h4, h5, h6 {
+  margin-top: 0;
+  margin-bottom: .5rem;
```

Comment:
`_normalize.scss` and `_reset.scss` don't seem to be used. Should we delete them for cleaniness?